sormas-e2e-tests/src/test/resources/features/sanity/web/Vaccination.feature

```diff
@@ -87,6 +87,229 @@ Feature: Vaccination tests
     And I check if generated document for Event Participant based on "VaccinationGenerationTest_EventParticipants.docx" contains all required fields
     Then I delete downloaded file created from "VaccinationGenerationTest_EventParticipants.docx" Document Template for Event Participant
 
+  @issue=SORDEV-11570 @env_de
+  Scenario: Duplicate detection for vaccinations when merging cases[1]
```

Comment:
what's the difference between tests?
I can't understand what is with case [index]

Ping me on teams if is something more to explain :D maybe we can make it running with multiple parameters instead of having the same test multiple times